cid-redirects.json

```diff
@@ -3857,11 +3857,12 @@
   "/docs/api/beta": "/docs/api",
   "/docs/api/dashboard-data": "/docs/api/dashboard",
   "/docs/cloud-security-analytics": "/docs/cloud-infrastructure-security",
-  "/docs/cloud-security-analytics/introduction-to-cloud-security-analytics": "/docs/cloud-infrastructure-security/introduction-to-cloud-infrastructure-security",
-  "/docs/cloud-security-analytics/data-lake": "/docs/cloud-infrastructure-security/data-lake",
-  "/docs/cloud-security-analytics/audit-and-compliance": "/docs/cloud-infrastructure-security/audit-and-compliance",
-  "/docs/cloud-security-analytics/threat-detection-and-investigation": "/docs/cloud-infrastructure-security/threat-detection-and-investigation",
-  "/docs/cloud-security-analytics/application-security": "/docs/cloud-infrastructure-security/application-security",
+  "/docs/cloud-security-analytics/introduction-to-cloud-security-analytics": "/docs/security",
+  "/docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws": "/docs/security",
+  "/docs/cloud-security-analytics/data-lake": "/docs/security",
+  "/docs/cloud-security-analytics/audit-and-compliance": "/docs/security",
+  "/docs/cloud-security-analytics/threat-detection-and-investigation": "/docs/security",
+  "/docs/cloud-security-analytics/application-security": "/docs/security",
```

Comment:
Are these redirects incomplete?

Reply:
Yes, deliberately so. If I put in the full new path now, it breaks the PR build. Once the PR is published, and the articles are available at the new paths, I'll provide the full paths in the redirect.